package.json

```diff
@@ -22,8 +22,10 @@
 		"element-ready": "^3.0.0",
 		"github-injection": "^1.0.1",
 		"github-reserved-names": "^1.0.14",
+		"graphql-request": "^1.6.0",
 		"linkify-issues": "^1.3.0",
 		"linkify-urls": "^2.0.0",
+		"lodash": "^4.17.10",
```

Comment:
The seems overkill, if we **need** some of the functions, can we bring in some of them instead of the entire library? We previously had it and removed it.